CONTRIBUTING.md

```diff
@@ -0,0 +1,40 @@
+# Contributing to this repository
+
+
+
+### Notes
+- ```master``` branch is always the 'current' branch where development for the next minor or major release is taking place
+- named branches (e.g. ```12.1```, or ```13.2```) are release branches where releases are tagged
+- All branches have gating enabled, and patches should not be merged that do not pass gating
+- QE testing is only performed on major (e.g. ```12.0.0```) and minor (e.g. ```12.1.0```) releases, not on patch (e.g. ```12.1.5```) releases, which are assumed to be adequately tested by the commit based, and periodic, jenkins gating.
+
+## fixing a bug
+
+1. A bug is filed or feature requested against ```rpc-openstack```
+2. After triage and prioritisation, the developer pushes a fix directly to a branch on ```rpc-openstack``` in the ```rcbops``` github namespace (rather than a developers own fork). A pull request is then made from this branch to the ```master``` branch.
+3. Unless a bug is specific to a release branch (e.g. ```12.1```), fixes are made to the ```master``` branch before any potential backports.
+4. Bugs meeting backport criteria are backported to the release branches (e.g. ```12.1```) as appropriate.
+5. github markdown is used to update the original issue with a checklist for tracking which branches have had fixes merged.
+6. Each time a PR is merged, the associated branch is deleted from ```rpc-openstack```.
```

Comment:
Should we define what the criteria is for a review getting accepted? I think right now there's an implicit understanding that a review needs two +1s from people currently on the RPC engineering team. However, this is not actually stated anywhere that I know of, which would be helpful in setting a standard and communicating it outward.